compiler/rustc_hir_typeck/src/method/mod.rs

```diff
@@ -332,8 +332,8 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
 
         // Construct an obligation
         let poly_trait_ref = ty::Binder::dummy(trait_ref);
-        let output_ty = expected.only_has_type(self).and_then(|ty| (!ty.needs_infer()).then(|| ty));
 
+        let output_ty = expected.only_has_type(self).and_then(|ty| (!ty.needs_infer()).then(|| ty));
```

Comment:
unrelated change?

Reply:
github diff shout is wrong? it is the same code